CHANGELOG.md

```diff
@@ -8,20 +8,53 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 
+### Removed
+
+* Removed [replace_with], [static_assertions], and [sealed] from the crate dependencies. ([#353])
```

Comment:
This should be moved to `Changed` since it's a purely internal change. `Removed` should only be used for public APIs that have been deleted; otherwise, the reader might mistakenly assume this is a breaking change.